app/Rules/Valid.php

```diff
@@ -88,13 +91,34 @@ public static function datetime(
         bool $sometimes = false,
         bool $nullable = false,
     ): array {
-        return self::base($sometimes, $nullable, ['string', 'date_format:Y-m-d\\TH:i:s\\Z'], $rules);
+        return self::base($sometimes, $nullable, ['string', 'date_format:Y-m-d\\TH:i:sp'], $rules);
```

Comment:
chodzi o przepuszczenie tylko dat zakończonych Z (wymuszenie offsetu +00:00)

`date_format:Y-m-d\\TH:i:sp` przepuści 2023-11-13T20:12:13+01:00

Reply:
można to dopisać w komentarzu tutaj pewnie

Reply:
Zależy nam na tym, żeby to było w Zulu?

Reply:
tak, bo w przeciwnym razie trzeba znaleźć inne miejsce w którym backend będzie pilnował offsetu strefy czasowej. Bardzo łatwo uzyskać błąd tego typu, że 12:00:00+02:00 zostanie zapisana do bazy jako 12:00:00 i odczytana jako 12:00:00+00:00

Reply:
Jasne, cofam tę zmianę.